types/rpmname.pp

```diff
@@ -0,0 +1,4 @@
+# Valid RPM names can be almost anything
+# Example openssh
+# Example python35-foobar
```

Comment:
Looks like this it interpreted and causes the odd rendering. Not sure what the best solution is.